<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="3">
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="vat_lamp_normsty">
        <property name="text">
         <string>NORMSTY</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="vat_lamp_luftsys">
        <property name="text">
         <string>LUFTSYS</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="vat_lamp_likstrm">
        <property name="text">
         <string>LIKSTRM</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="vat_lamp_hhp1">
        <property name="text">
         <string>H HP 1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="vat_lamp_oljetr">
        <property name="text">
         <string>OLJETR</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="vat_lamp_resgen">
        <property name="text">
         <string>RESGEN</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="vat_lamp_hgen">
        <property name="text">
         <string>H GEN</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="vat_lamp_dragkr">
        <property name="text">
         <string>DRAGKR</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="vat_lamp_motor">
        <property name="text">
         <string>MOTOR</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="vat_lamp_brasys">
        <property name="text">
         <string>BRÄSYS</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="vat_lamp_abumod">
        <property name="text">
         <string>ABU-MOD</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="vat_lamp_styrsak">
        <property name="text">
         <string>STYRSÄK</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="vat_lamp_felinfo">
        <property name="text">
         <string>FELINFO</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="vat_lamp_primdat">
        <property name="text">
         <string>PRIMDAT</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="vat_lamp_uppdrag">
        <property name="text">
         <string>UPPDRAG</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="vat_lamp_dator">
        <property name="text">
         <string>DATOR</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="vat_lamp_hydr1">
        <property name="text">
         <string>HYDR 1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="vat_lamp_hydr2">
        <property name="text">
         <string>HYDR 2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="vat_lamp_streck2">
        <property name="text">
         <string>----</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="vat_lamp_mobrand">
        <property name="text">
         <string>MOBRAND</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="vat_lamp_landst">
        <property name="text">
         <string>LANDST</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="vat_lamp_bramgd">
        <property name="text">
         <string>BRÄMGD</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="vat_lamp_apu">
        <property name="text">
         <string>APU</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="vat_lamp_bromsar">
        <property name="text">
         <string>BROMSAR</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLabel" name="vat_lamp_oxykab">
        <property name="text">
         <string>OXY/KAB</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="vat_lamp_apubrnd">
        <property name="text">
         <string>APUBRND</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="vat_lamp_streck1">
        <property name="text">
         <string>-----</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QLabel" name="vat_lamp_huvstol">
        <property name="text">
         <string>HUVSTOL</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="4">
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="6" column="1">
       <widget class="QPushButton" name="button_ess_on">
        <property name="text">
         <string>ESS</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="button_falltank">
        <property name="text">
         <string>Fällning Tank</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="button_hstrom_on">
        <property name="text">
         <string>HSTROM</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QPushButton" name="button_lamptest">
        <property name="text">
         <string>LampTest</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="button_lt_kran_on">
        <property name="text">
         <string>LT KRAN</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QPushButton" name="button_systest">
        <property name="text">
         <string>SysTest</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QGridLayout" name="gridLayout_8">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="2">
         <widget class="QPushButton" name="button_apu_on">
          <property name="text">
           <string>APU</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="lamps_apu_red">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lamps_apu_gar">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="5">
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="button_sand">
        <property name="text">
         <string>SÄND</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="button_mkv">
        <property name="text">
         <string>MKV</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="button_rhm">
        <property name="text">
         <string>RHM</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="button_reload_acf">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>100</horstretch>
          <verstretch>100</verstretch>
         </sizepolicy>
        </property>
        <property name="baseSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>RÄDDA PLANET</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="7" column="3">
       <widget class="QPushButton" name="button_spak">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>SPAK</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="button_hojd">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>HOJD</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lamps_master1">
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Huvud</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="lamps_gears">
        <property name="text">
         <string>Landställ</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="text_fuel">
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="QLabel" name="lamps_hojdvarn">
        <property name="text">
         <string>HOJDVARN</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="lamps_a14">
        <property name="text">
         <string>A14</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QSpinBox" name="auto_afk_text">
        <property name="maximum">
         <number>3000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="button_tanka_50">
        <property name="text">
         <string>Tanka 50%</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Bränsle</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="button_start">
        <property name="text">
         <string>Motor Start</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="button_tanka">
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Tanka Full</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="button_afk">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>AFK</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="button_master">
        <property name="text">
         <string>Kvittera Larm</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="lamps_ks">
        <property name="text">
         <string>KS</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QSpinBox" name="auto_hojd_text">
        <property name="maximum">
         <number>30000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QPushButton" name="button_att">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>ATT</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lamps_master2">
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Varning</string>
        </property>
       </widget>
      </item>
      <item row="7" column="5">
       <widget class="QLabel" name="lamps_park">
        <property name="text">
         <string>Parkeringsbroms</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="mfdButton11">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="mfdButton12">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLabel" name="lamps_airbrake">
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Luftbroms</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="2" column="2">
       <widget class="QLabel" name="st_lamp_s">
        <property name="text">
         <string>S</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="st_lamp_j">
        <property name="text">
         <string>J</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="st_lamp_a">
        <property name="text">
         <string>A</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
